packages/fuels-core/src/code_gen/abigen.rs

```diff
@@ -43,7 +43,7 @@ impl Abigen {
         Ok(Self {
             types: Abigen::get_types(&parsed_abi),
             abi: parsed_abi,
-            contract_name: ident(contract_name),
+            program_name: ident(contract_name),
```

Comment:
I would just use `name`.. No need to make it confusing with `program`
```suggestion
            name: ident(contract_name),
```